### ISSUE TITLE
RMARTS format error

### ISSUE THREAD

**Issue by EiRamMeyre** [louadi/NEASE#4]
Dear Louadi,

For our project, we want to compare the number of splicing events obtained by MAJIQ with RMARTS. Then, we want to see the differences in results in NEASE between these two tools but there is a format issue on the RMARTS table. We get the following error: "process failed....Try to use the Standard input". We have looked at the code and we have the required columns, I attach the rmarts data. Can you tell me what exactly the expected format is?
[SE.MATS.JCEC.txt](https://github.com/louadi/NEASE/files/8196238/SE.MATS.JCEC.txt)
[SE.MATS.JC.txt](https://github.com/louadi/NEASE/files/8196248/SE.MATS.JC.txt)

Thank you for your help,

Marie

**Comment by louadi** [comment]
Hi Marie,

Thanks for opening the issue.

I manage to replicate your error and it seems like the problem was on the gene id that includes the version in your table. For now, you can go around it by:


```python
data=pd.read_table('SE.MATS.JCEC.txt')
data=data[['GeneID','exonStart_0base','exonEnd','IncLevelDifference']]
data['GeneID']=data['GeneID'].apply(lambda x: x.split('.')[0])

events=nease.run(data, organism='Human')

```

I manage to run it this way. I will also fix the bug for future users.

Best regards,
Zakaria

**Comment by louadi** [comment]
Fixed - 490fae6667bb5500fc7f5e69338611d839048050

**Comment by EiRamMeyre** [comment]
Hi Zakaria,

Indeed that was it, thank you for your quick response.

Best regards,

Marie
